src/gridcoin/researcher.cpp

```diff
@@ -794,18 +794,23 @@ MiningProject::MiningProject(std::string name,
     , m_rac(std::move(rac))
     , m_error(Error::NONE)
 {
-    boost::to_lower(m_team);
+    m_team = ToLower(m_team);
 }
 
 MiningProject MiningProject::Parse(const std::string& xml)
 {
+    double rac = 0.0;
+
+    if (!::ParseDouble(ExtractXML(xml, "<user_expavg_credit>", "</user_expavg_credit>").c_str(), &rac)) {
+        rac = 0.0;
```

Comment:
The newest version removes the unnecessary assignment of 0.0, since that is the behavior of ParseDouble. Now a warning is put in the log and it continues. Continuing is the original behavior, I believe, because the original was strtod, which was used without paying attention to the errno. The documentation says that strtod returns a 0.0, which is the behavior here. Given that we are talking about this function being activated for legacy only data in the historical chain, I do not want to change the behavior as it could cause a sync from zero problem.